redbiom/util.py

```diff
@@ -368,3 +368,19 @@ def stems(stops, stemmer, string):
             yield stemmer.stem(word).lower()
         except Exception:
             continue
+
+
+def biom_md5_feature_ids(table, output):
```

Comment:
This method should not write a file as it's an internal call. As written, users of the Python API cannot use this method. Can ID mapping either be encoded within the BIOM table or written out within the CLI?

Also, would it be possible to add in a numpydoc string?